packages/server/lib/reporter.js

```diff
@@ -143,7 +154,7 @@ const mergeRunnable = (eventName) => {
         const prevAttempts = runnable.prevAttempts || []
 
         delete runnable.prevAttempts
-        const prevAttempt = _.cloneDeep(runnable)
+        const prevAttempt = toAttemptProps(runnable)
```

Comment:
What was the original reason for deep-cloning this? Are we worried about mutations? If so, it still would be possible to mutate `err` and `timings`.